### PR TITLE
Prototech 45: Revert after instead of on `expiry_`

### PR DESCRIPTION
**Pull request by prat-gpt**
# Description of change
## High level
* See - https://github.com/Fixed-Point-Solutions/prototech-ajna-audit/issues/45
* Update transaction revert only after expiry time rather than on and after expiry time.

### CODE REVIEW

**Review by grandizzy — COMMENTED** [review]
hm, I see many places in our test where `block.timestamp + 1` is passed, wonder why tests doesn't fail?

**Comment by grandizzy** [comment]
> hm, I see many places in our test where `block.timestamp + 1` is passed, wonder why tests doesn't fail?

pls disregard, my bad

**Review by grandizzy — APPROVED** [review]
LGTM

File: src/libraries/helpers/RevertsHelper.sol [hunk 1]
```diff
@@ -84,7 +84,7 @@ import { Maths }    from '../internal/Maths.sol';
     function _revertOnExpiry(
```
Comment (MikeHathaway) [inline src/libraries/helpers/RevertsHelper.sol]: Should we also rename this method?

**Review by MikeHathaway — APPROVED** [review]
LGTM - just one comment about potentially renaming the method

File: src/libraries/helpers/RevertsHelper.sol (hunk 1 above)
Comment (EdNoepel) [reply src/libraries/helpers/RevertsHelper.sol]: If trivial to change, I agree `_revertAfterExpiry` would be better now.

**Review by EdNoepel — APPROVED** [review]

File: src/libraries/helpers/RevertsHelper.sol (hunk 1 above)
Comment (grandizzy) [reply src/libraries/helpers/RevertsHelper.sol]: rename in https://github.com/ajna-finance/contracts/pull/859/commits/8e7540ab8fed928984ec39f7059be830c03f687a

**Pull request merged**